kolibri/content/models.py

```diff
@@ -189,7 +199,12 @@ def save(self, *args, **kwargs):
                 content_copy_track.referenced_count -= 1
                 content_copy_track.save()
                 if content_copy_track.referenced_count == 0:
-                    content_copy_path = os.path.join(settings.CONTENT_COPY_DIR, self.checksum[0:1], self.checksum[1:2], self.checksum + self.extension)
+                    content_copy_path = os.path.join(
+                        settings.CONTENT_COPY_DIR,
+                        self.checksum[0:1],
+                        self.checksum[1:2],
+                        self.checksum + '.' + self.file_format.extension
+                    )
                     if os.path.isfile(content_copy_path):
                         os.remove(content_copy_path)
```

Comment:
Ah, I see, it's when `referenced_count` is `0`.

I think one thing we need to do is revisit the high-level discussion about the sharing of this code between the curation app and the installable app, as the needs are actually quite different.